_docs/setup/kubernetes/mesh-expansion.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Mesh Expansion
+overview: Instructions to add external machines and expanding Istio.
+
+order: 60
+
+layout: docs
+type: markdown
+---
+
+Instructions to configure Istio on a Kubernetes Cluster so it can be expanded with
+services running on cloud or on-prem VMs or external machines.
+
+## Prerequisites
+
+* You have [installed Istio](quick-start.html) on Kubernetes.
+
+* The machine must have IP connectivity to the endpoints in the mesh. This
+typically requires same VPC or a VPN connection, as well as a container network that
```

Comment:
the same

Reply:
removing same altogether - it should be (but too long) "requires being inside the same VPC, or a VPN (connection)" but I'm cutting /simplifying